packages/at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart

```diff
@@ -48,136 +50,152 @@ class _AtOnboardingAccountsScreenState
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          'Select atSigns',
-          style: TextStyle(
-            color: Platform.isIOS || Platform.isAndroid
-                ? Theme.of(context).brightness == Brightness.light
-                    ? Colors.black
-                    : Colors.white
-                : null,
+    final theme = Theme.of(context).copyWith(
+      primaryColor: widget.config.theme?.primaryColor,
+      colorScheme: Theme.of(context).colorScheme.copyWith(
+            primary: widget.config.theme?.primaryColor,
```

Comment:
Are we only using `primary ` data from the set theme ?

Reply:
Yes